app/assess/data.py

```diff
@@ -58,6 +50,15 @@ def get_local_data(endpoint: str):
         return api_data.get(endpoint)
 
 
+def call_search_applications(params: dict):
+    applications_endpoint = (
+        APPLICATION_STORE_API_HOST
+        + APPLICATIONS_SEARCH_ENDPOINT.format(params=urlencode(params))
```

Comment:
I haven't seen this before but it would actually be really useful to use. I'll try and use url encode for url args from now :) good one!